plugins/chartbeat.client.js

```diff
@@ -0,0 +1,23 @@
+export default defineNuxtPlugin(() => {
+  /* eslint-disable */
+  (function() {
+    /** CONFIGURATION START **/
+    const _sf_async_config = window._sf_async_config = (window._sf_async_config || {});
+    _sf_async_config.uid = 16246; //CHANGE THIS TO YOUR ACCOUNT ID
```

Comment:
should this ID be an env var?

Reply:
It doesn't change for demo or prod - but this is how it was setup on the old site (I didn't set it up)

Reply:
security?